src/NServiceBus.ContainerTests/When_using_nested_containers.cs

```diff
@@ -135,8 +113,41 @@ public void Instance_per_call_components_should_not_cause_memory_leaks()
                 var upperLimitBytes = 200*1024;
                 Assert.That(after - before, Is.LessThan(upperLimitBytes), "Apparently {0} consumed more than {1} KB of memory", builder, upperLimitBytes/1024);
             }
+        }
+
+        [Test]
+        [Explicit]
+        public void Instance_per_uow_components_should_not_cause_memory_leaks()
```

Comment:
If the test below (`UoW_components_in_the_parent_container_should_be_singletons_in_the_child_container`) checks that UOW instances in child containers are singletons from the parent containers, does this test actually do anything since the child container will just be returning the same instance every time?